sys-apps/shadow/shadow-4.14.6-r1.ebuild

```diff
@@ -45,14 +45,6 @@ DEPEND="
 "
 RDEPEND="
 	${COMMON_DEPEND}
-	!<sys-apps/man-pages-5.11-r1
```

Comment:
I think this is fine at this point, but note that we try to keep blockers for ~2 years after the last match is gone to help people upgrading old systems.